src/App.jsx

```diff
@@ -13,6 +15,8 @@ function App() {
         <Route path="/" element={<Home></Home>}></Route>
         <Route path="/articles" element={<Articles></Articles>}></Route>
         <Route path="/articles/:article_id" element={<Article></Article>}></Route>
+        <Route path="/topics" element={<Topics></Topics>}></Route>
+        <Route path="/topics/:topic_name" element={<Topic></Topic>}></Route>
```

Comment:
nicely done! It's always nice to be able to copy + paste links directly to a filtered page I find